### ISSUE TITLE
<nameEntryParallel>

### ISSUE THREAD

**Issue by SJagodzinski** [SAA-SDT/eac-cpf-schema#117]
## Name Entry Parallel
- **rename** to `<nameEntrySet>`
- **remove child elements** `<alternativeForm>`, `<authorizedForm>`
- keep availability
- remove xml namespace
- add **optional attributes**:
`@audience`
`@conventationDeclarationReference` 
`@languageOfElement`
`@localTypeDeclarationReference`
`@maintenanceEventReference` 
`@scriptOfElement`
`@sourceReference`
- specify **Tag Library** and **Best Practise Guide** for usage and examples for parallel names

## Creator of issue
1. Silke Jagodzinski
2. TS-EAS: EAC-CPF subgroup
3. silkejagodzinski@gmail.com

## Related issues / documents
Names control #26 
`<nameEntryParallel>`: rename #66 
`<nameEntryParallel>` - revise description in the Tag Library #56 
[Topic_names_20200130.pdf](https://github.com/SAA-SDT/TS-EAS-subteam-notes/blob/master/eaccpf-subteam/working-documents/topics/names/Topic_names_20200130.pdf)
New attribute @status #70 

## EAD3 Reconciliation
EAC-CPF specific element

## Context
A wrapper element for two or more `<nameEntry>` elements that represent parallel forms of the name (e.g., official forms of the name in different languages and/or scripts, transliterated forms of the name).

**May contain**: `<alternativeForm>` , `<authorizedForm>` , `<nameEntry>` , `<useDates>`
**May occur within**: `<identity>`
**Attributes:** `@localType`, `@xml:id`
**Availability:** Optional, Repeatable

## Solution documentation
Rephrasing _Summary_, _Description and Usage_ and _Attribute usage_ needed

**May contain**: `<nameEntry>` , `<useDates>`
**May occur within**: `<identity>`
**Attributes:** 
`@audience` - optional (values limited to: external, internal)
`@conventationDeclarationReference` - optional
`@id` - optional
`@languageOfElement` - optional 
`@localType` - optional
`@localTypeDeclarationReference` - optional
`@maintenanceEventReference` - optional
`@scriptOfElement` - optional
`@sourceReference` - optional
**Availability:** Optional, Repeatable

- New or other example needed
- Topic for Best Practise Guide:
> Use the optional attribute `@localType` to reflect the usage of parallel names in North America.

> There is obviously a different interpretation according to this element in the different traditions of archival description in Europe and North America.

> Explain that the element is meant to encode parallel names like translations or pseudonyms, as the example shows, and not changed names like birth names or maiden names. Add examples containing with translations/transliterations and another exampel for names in differents periods of someones life.

## Example encoding
```
<identity>
 <entityType value="family"/>
 <nameEntrySet audience="external" conventionDeclarationReference="conventiondeclaration1" id="nameEntrySet1" languageOfElement="en" localType="localNameEntryset" localTypeDeclarationReference="localTypeDeclaration1" maintenanceEventReference="maintenancevent1" scriptOfElement="lat" sourceReference="source1">
   <nameEntry></nameEntry>
   <nameEntry></nameEntry>
   <useDates></useDates>
 </nameEntrySet
</identity>
```

**Comment by SJagodzinski** [comment]
@wisserk per E-Mail, 9 March 2020:

>  I did look into the history of the use of the term "parallel" and it seems to have appeared in the first edition of the International Standard for Bibliographic Description 1972. In the current edition of RDA, it uses the term "parallel" for a variety of "parallel" fields all with the same basic definition: "in a language or script that differs from".

> I do think that maintaining the parallel only for language issues (in the cases I have shown in the examples attached) and establishing a variant name element for the use cases you all described in Austin is the best course of action.

**Comment by kerstarno** [comment]
I'm not sure, I understand what the proposal is exactly following Kathy's email. 

Does "maintaining the parallel only for language issues" mean, we should revert back to naming the element `<nameentryparallel>`, using this specifically for translations and transcriptions of the name(s), while creating an *additional* "variante name element" (e.g. `<nameentryset>`) for other use cases?

Or does it suggest to mainly keep naming the element `<nameentryparallel>` and using the `@localtype` for distinction as to whether the parallel name(s) is/are translations(s), transcription(s) or other variant(s)?

Or is it still ok with moving towards the initial proposal of this issue here, i.e. renaming to `<nameentryset>` and using the `@localtype` for distinction as to whether the parallel name(s) is/are translations(s), transcription(s) or other variant(s)?

**Comment by kerstarno** [comment]
Sorry for backtracking on this one, but I'm currently looking at the new way of encoding names in preparation of parallel conversations on the topic for EAG. 

I'm not sure, we actually discussed this and I couldn't find it specifically mentioned in the topic paper, but seeing that `<nameEntryParallel>` (now: `<nameEntrySet>`) includes the sub-elements `<authorizedForm>` and `<alternativeForm>` as well, wouldn't we use `@status`(#70) with values "authorized" and "alternative" also with `<nameEntrySet>`?

**Comment by SJagodzinski** [comment]
@kerstarno : We will use `@status` with values "authorized" and "alternative" to indicate the status of a name entry, not for the whole name entry set. The idea is to bundle translated or transcribed name entries and they do not have be authorized or alternative all together. If so, they all can get the same status value. But looking at the use cases we discussed, the same status for all name entries might be an exception.

**Comment by kerstarno** [comment]
@SJagodzinski - thanks for the clarification. I had only been wondering as I couldn't seem to find this written down anywhere. So, `<authorizedForm>` and `<alternativeForm>` will be removed from `<nameEntryParallel>` (now: `<nameEntrySet>`), but not be replaced by the according attributes, which will instead only be used for the constituent `<nameEntry>`-s that are group within the `<nameEntrySet>`. 

Fine with me. Good to note probably for the transformation script from current EAC-CPF to EAC-CPF 2.0 (@fordmadox).

**Comment by ailie-s** [comment]
Draft Tag Library Text:

**Summary:** A wrapper element for two or more `<nameEntry>` elements representing different forms of the same name (e.g. official forms of the name in different languages and/or scripts)
**May contain:** nameEntry (2..n), useDates (0..1)
**May occur within:** identity
**Attributes:**
`@audience` - optional (values limited to: external, internal)
`@conventationDeclarationReference` - optional
`@id` - optional
`@languageOfElement` - optional
`@localType` - optional
`@localTypeDeclarationReference` - optional
`@maintenanceEventReference` - optional
`@scriptOfElement` - optional
`@sourceReference` - optional
**Description and Usage:** 
A wrapper element used to group two or more `<nameEntry>` elements representing parallel forms of the name for the same CPF entity which are used at the same time (e.g., official forms of the name in different languages and/or scripts).
The `<nameEntrySet>` element may contain a `<useDates>` element to indicate the dates the set of name forms was used.
**Availability:** 
Within `<identity>`: one of `<nameEntry>` or `<nameEntrySet>` required, repeatable

**Comment by karinbredenberg** [comment]
Test of element nameEntrySet:
May contain: nameEntry (2..n), useDates (0..1) **Ok in both schemas**
May occur within: identity **Ok in both schemas**
@audience - optional (values limited to: external, internal) **Ok in both schemas**
@conventationDeclarationReference - optional **Ok in both schemas**
@id - optional **Ok in both schemas**
@languageOfElement - optional **Missing in both schemas**
@localType - optional **Ok in both schemas**
@localTypeDeclarationReference - optional **Ok in both schemas**
@maintenanceEventReference - optional **Ok in both schemas**
@scriptOfElement - optional **Missing in both schemas**
@sourceReference - optional **Ok in both schemas**

**Result**: Addition of the two language attributes, languageOfElement and scriptOfElement needed.

**Comment by karinbredenberg** [comment]
Retested element nameEntrySet:
languageOfElement **Added in both schemas**
scriptOfElement **Added in both schemas**

**Result**: Element ready.

**Comment by fordmadox** [comment]
Note:  I previously defined useDates as "zero or more" here.  I'm changing that back to 0..1 now (update: not changing it back, since the useDates issue mentions the element is indeed repeatable).  However, I used 0..n before due to discussions around the assertion proposal, which might record conflicting date ranges.

**Comment by fordmadox** [comment]
Actually, useDates is listed as 0..n on its own issue page.  @ailie-s I think that means that the TL documentation (above) will need to be updated.

**Comment by SJagodzinski** [comment]
@fordmadox : Correct, we agreed to repeat `<useDates>`, cf. https://github.com/SAA-SDT/eac-cpf-schema/issues/134#issuecomment-724269034

**Comment by fordmadox** [comment]
Okay, so all that is left to figure out is the conversion process.  I'll have a first pass of that ready soon.  I think we might need to adjust things slightly with the `@conventionDeclarationReference` approach within nameEntry (and how that is optionally bundled in nameEntryParallel currently) just based on all the possible EAC encodings, but I haven't investigated enough yet to be certain.  

I'm assuming, so far, though, that there is not a need in EAC 2.0 to have a single nameEntry labelled as "authorized" in one or more sources, as well as labelled as "alternative" in one or more other sources?  But not sure, looking again at what's possible in EAC now.  

Then, there's the further complication of `@transliteration`.  I like the decision not to confine that to an attribute value alone, but even if we allowed `@conventionDeclarationReference` to store multiple IDREFS, I'm not exactly sure how that would be processed / useful.  I'd need to try it out first, I think, but right now just thinking through the conversion process. :smile: 

If anyone has already mapped this out, though, please let me know.

**Comment by kerstarno** [comment]
> Okay, so all that is left to figure out is the conversion process. I'll have a first pass of that ready soon. I think we might need to adjust things slightly with the `@conventionDeclarationReference` approach within nameEntry (and how that is optionally bundled in nameEntryParallel currently) just based on all the possible EAC encodings, but I haven't investigated enough yet to be certain.

As I am currently working my way through creating an EAC-CPF 1.0 sample file for conversion testing, this is how I'd see this part:
- If `<nameEntryParallel>` is used in EAC-CPF 1.0 and if this `<nameEntryParallel>` also includes either `<authorizedForm>` or `<alternativeForm>` for the complete set of `<nameEntry>`-s, 
  - Each `<nameEntry>` in the (converted) EAC-CPF 2.0 `<nameEntrySet>` would get the `@status` "authorized" respectively "alternative"
  - If, additionally, a single `<nameEntry>` within such `<nameEntryParallel>` includes the sub-element `<preferredForm>`, this `<nameEntry> would get assigned the `@preferredForm` "yes" in EAC-CPF 2.0
  - The content of `<authorizedForm>`, `<alternativeForm>`, and `<preferredForm>`, which is supposed to be the abbreviated name of "a set of national, international or other rules that govern the construction of names", would be moved to  `<conventionDeclaration><shortCode>` in the first place
    - Seeing that `<conventionDeclaration>` requires `<reference>`, we could either create an empty `<reference/>` as part of the conversion process or could consider repeating the content of `<authorizedForm>`, `<alternativeForm>`, and `<preferredForm>` in `<conventionDeclaration><reference>` as well along with a conversion comment that it is recommended to review this in both scenarios
  - As the status is now with each `<nameEntry>` rather than being applied to the whole set, the `@conventionDeclarationReference` for the rule that designated a name as "authorized" or "alternative" should - in my opinion - also be added to the `<nameEntry>`-s rather than the (converted) EAC-CPF 2.0 `<nameEntrySet>`
  - Latest at this point, we probably have to consider enabling multiple IDREF-s in @conventionDeclarationReference, because one `<nameEntry>` might point to one `<conventionDeclaration>` for the (inherited) status as "authorized" or "alternative" plus pointing to another `<conventionDeclaration>` for its (original) status of being a "preferred" form of name
  - Furthermore, all three elements - `<authorizedForm>` and `<alternativeForm>`, on the EAC-CPF 1.0 level of `<nameEntryParallel>`, and `<preferredForm>`, on the EAC-CPF 1.0 level of a single `<nameEntry>` - can be repeated e.g. to indicate a second set of rules that supports the status designation; here we'd again need an option for multiple IDREF-s in `@conventionDeclarationReference`
- If instead several `<nameEntry>`-s are used in EAC-CPF 1.0 and if some of these also include either `<authorizedForm>` or `<alternativeForm>`,
  -  Each `<nameEntry>` would get the appropriate `@status` "authorized" respectively "alternative" in EAC-CPF 2.0
  -  The content of `<authorizedForm>` respectively `<alternativeForm>` would again be moved to  `<conventionDeclaration><shortCode>` in the first place, either with an empty `<reference/>` or with `<conventionDeclaration><reference>` repeating the same content
    - `<preferredForm>` does not play a role in this case, as it's only available with `<nameEntry>` when grouped within `<nameEntryParallel>` in EAC-CPF 1.0
    - However,  `<authorizedForm>` respectively `<alternativeForm>` again are repeatable e.g. to indicate a second set of rules that supports the status designation; so we'd again need an option for multiple IDREF-s in `@conventionDeclarationReference` in order to point to the `<conventionDeclaration>` that has been converted from the content of `<authorizedForm>` respectively `<alternativeForm>`


> I'm assuming, so far, though, that there is not a need in EAC 2.0 to have a single nameEntry labelled as "authorized" in one or more sources, as well as labelled as "alternative" in one or more other sources? But not sure, looking again at what's possible in EAC now.

Personally, I wouldn't mix "sources" and "rules" in connection with the EAC-CPF 1.0 elements `<authorizedForm>`, `<alternativeForm>`, and `<preferredForm>` for their conversion to EAC-CPF 2.0, but would stick with "rules" and hence with `<conventionDeclaration>` and `@conventionDeclarationReference` only. That being said, from an EAC-CPF 2.0 perspective it might nonetheless be useful considering the option of multiple IDREF-s for `@sourceReference` as well, because especially in disputed cases one might want to cite a supporting source in addition to the original source of one's assertion.


> Then, there's the further complication of `@transliteration`. I like the decision not to confine that to an attribute value alone, but even if we allowed `@conventionDeclarationReference` to store multiple IDREFS, I'm not exactly sure how that would be processed / useful. I'd need to try it out first, I think, but right now just thinking through the conversion process. 😄
> 
> If anyone has already mapped this out, though, please let me know.

I think, this might mainly be a question in case we end up with having multiple IDREF-s for different "purposes" so to speak. And it might then be bound to the question, whether we actually still want/need to distinguish these different purposes or if the main aspect is the mentioning of the rule independent of whether that's to assign the status of "authorized", "alternative", or "preferred" or whether it's to indicate the transliteration scheme. 

E.g. an EAC-CPF 1.0 `<nameEntry>` with the following encoding:
``` 
<identity>
  [...]
  <nameEntryParallel>
    <nameEntry transliteration="DIN32708:2014">
      <part>Some name</part>
      <preferredForm>AFNOR</preferredForm>
    </nameEntry>
    <nameEntry>
      <part>Another name</part>
    </nameEntry>
    <authorizedForm>VIAF</authorizedForm>
  </nameEntryParallel>
  [...]
</identity>
```
could become
```
<control>
  [...]
  <conventionDeclaration id="cd1">
    <shortCode>DIN32708:2014</shortCode>
    <reference/>
  </conventionDeclaration>
  <conventionDeclaration id="cd2">
    <shortCode>AFNOR</shortCode>
    <reference/>
  </conventionDeclaration>
  <conventionDeclaration id="cd3">
    <shortCode>VIAF</shortCode>
    <reference/>
  </conventionDeclaration>
  [...]
</control>
[...]
<identity>
  [...]
  <nameEntrySet>
    <nameEntry conventionDeclarationReference="cd1 cd2 cd3" status="authorized" preferredForm="true">
      <part>Some name</part>
    </nameEntry>
    <nameEntry conventionDeclarationReference="cd3" status="authorized">
      <part>Another name</part>
    </nameEntry>
  </nameEntrySet>
  [...]
</identity>
```

**Comment by fordmadox** [comment]
@SJagodzinski But the drawback with that approach is that there is no way to distinguish between the transliteration scheme and the name formation rules, correct?  I think that renders the transliteration part slightly useless, but I could be wrong.  There would be no reason, though, that another attribute couldn't also point to a convention declaration (e.g. transliterationReference :smile:)

**Comment by fordmadox** [comment]
Also, just to be clear: right now, the reference attributes are defined as IDREF.  I can change all of those to IDREFS so that the above is possible, if that's desired.  

For the first part of the transformation process, I ensured that only one value would be added to `@conventionDeclarationReference`, but for nameEntry, it would be preferable to open that open (as illustrated in the above example) so that IDREFS were possible.

For the time being, at least, I have changed all IDREF attributes to IDREFS so that we can support the above example, @SJagodzinski.  What does everyone think of this approach: https://github.com/SAA-SDT/eac-cpf-schema/commit/d67bb5d1c6a9faa71e42170d3a98f33773676418.

**Comment by kerstarno** [comment]
> But the drawback with that approach is that there is no way to distinguish between the transliteration scheme and the name formation rules, correct? 

Yes, indeed. As mentioned above. 

The question, though, is: do we want/need to be able to do that? In both cases, we are referencing a rule that was used in forming and/or categorising the element's content, whether that is for the formation of the name or for the transliteration of the name from a different writing system. Plus: there could be other types of rules that we might want to point to in the context of names and we don't distinguish these either, e.g. if I wanted to simply state that the fact that I am providing a name is following the recommendations of ISAAR-CPF.

**Comment by fordmadox** [comment]
>   * Each `<nameEntry>` would get the appropriate `@status` "authorized" respectively "alternative" in EAC-CPF 2.0

I still need to re-write the transformation to utilize IDREFS instead of IDREF, but I don't understand the above rule.  

`@status` can only have one value (authorized OR alternative).  If a nameEntry is flagged as both in EAC 1.0, that should result in what?  There's no way to bind the authorized reference to the nameEntry.  E.g. if something like this were the input:

```
            <nameEntry scriptCode="Latn" transliteration="some-scheme">
                <part localType="100a">Pope, Alexander, </part>
                <part localType="100d">1688-1744 </part>
                <authorizedForm>lcnaf</authorizedForm>
                <alternativeForm>some-other-thing</alternativeForm>
                <alternativeForm>yet-another-thing</alternativeForm>
                <authorizedForm>why-not-another</authorizedForm>
            </nameEntry>
```

What should the output be?

**Comment by fordmadox** [comment]
For the time being, since I'm assuming that status can only be 'authorized' or 'alternative', I'm preferring 'authorized' when present.  And, I've now updated the transformation to permit IDREFS, as per the example above.  So, the transformation would look something like:

```
         <nameEntry scriptOfElement="Latn"
                    status="authorized"
                    conventionDeclarationReference="cd1-transliteration cd2 cd3 cd4 cd5">
                     <!-- transliteration="afnor"-->
            <part localType="100a">Pope, Alexander, </part>
            <part localType="100d">1688-1744 </part>
            <!--
<authorizedForm>lcnaf</authorizedForm>
<alternativeForm>some-other-thing</alternativeForm>
<alternativeForm>yet-another-thing</alternativeForm>
<authorizedForm>why-not-another</authorizedForm>
-->
         </nameEntry>
```

**Comment by kerstarno** [comment]
Yes, sorry, I was assuming that a `<nameEntry>` respectively `<nameEntryParallel>` would either have `<authorizedForm>` or `<alternativeForm>` rather than having both in parallel. But, of course, this would be possible in EAC-CPF 1.0.

I think, I'd also tend to use "authorized" preferably in cases when both exist. Having the original input as comments in the converted file, would still allow users to change the status to "alternative", if wanted. 

Another option to deal with additional `<alternativeForm>`-s next to one or more `<authorizedForm>`-s could be to maybe still create a `<conventionDeclaration>` for each of these and add a default text in the `<descriptiveNote><p>` saying something like "This set of rules classifies the name as an alternative name." (or similar). And we could use `@target` and `@id` to link the `<conventionDeclaration>` back to the `<nameEntry>` so that it is clear which name is meant. E.g.  

```
<conventionDeclaration id="cd3" target="nE1">
  <shortCode>some-other-thing</shortCode>
  <reference/>
  <descriptiveNote>
    <p>This set of rules classifies the name as an alternative name.</p>
  </description>
</conventionDeclaration>
[...]
<nameEntry id="nE1" scriptOfElement="Latn" status="authorized" conventionDeclarationReference="cd1-transliteration cd2 cd3 cd4 cd5">
  <!-- transliteration="afnor"-->
  <part localType="100a">Pope, Alexander, </part>
  <part localType="100d">1688-1744 </part>
  <!--
    <authorizedForm>lcnaf</authorizedForm>
    <alternativeForm>some-other-thing</alternativeForm>
    <alternativeForm>yet-another-thing</alternativeForm>
    <authorizedForm>why-not-another</authorizedForm>
  -->
</nameEntry>
```

**Comment by kerstarno** [comment]
On another topic: I'd avoid using MARC21 codes as example values for `@localType`. If we wanted to have encodings in related standards enabled in EAC-CPF, we should have gone with introducing `@encodingAnalog` (and `@relatedEncoding`) as available in EAD ;-)

**Comment by fordmadox** [comment]
Regarding the conventionDeclaration/descriptiveNote approach...  how would that work when a single source could be referenced for both "authorized" and "alternative" names?  E.g. "some-other-thing" could be referenced elsewhere as housing the "authorizedForm" for another name in this file.

As for the other topic: minus the extra authorized/alternative/transliteration bits, that example is from a real-world file :smile: (see https://wiki.harvard.edu/confluence/display/connectingdots/Records?preview=/137171847/138018933/Pope.xml).  I still don't think that metadata mappings belong in each and every file / XML fragment, though, especially if it's limited to one mapping.

**Comment by kerstarno** [comment]
We might be getting more specific and sophisticated than necessary, but how about:
```
<conventionDeclaration id="cd3">
  <shortCode>some-other-thing</shortCode>
  <reference/>
  <descriptiveNote>
    <p target="nE1 nE3">This set of rules classifies certain names as alternative forms of names.</p>
    <p target="nE2">This set of rules classifies certain names as authorized forms of names.</p>
  </description>
</conventionDeclaration>
[...]
<nameEntry id="nE1" scriptOfElement="Latn" status="authorized" conventionDeclarationReference="cd1-transliteration cd2 cd3 cd4 cd5">
  <!-- transliteration="afnor"-->
  <part>Pope, Alexander, </part>
  <part>1688-1744 </part>
  <!--
    <authorizedForm>lcnaf</authorizedForm>
    <alternativeForm>some-other-thing</alternativeForm>
    <alternativeForm>yet-another-thing</alternativeForm>
    <authorizedForm>why-not-another</authorizedForm>
  -->
</nameEntry>
<nameEntry id="nE2" status="authorized" conventionDeclarationReference="cd2">
  <part>Another form of name</part>
  <!--
    <authorizedForm>some-other-thing</authorizedForm>
   -->
</nameEntry>
<nameEntry id="nE3" status="alternative" conventionDeclarationReference="cd2">
  <part>A third form of name</part>
  <!--
    <alternativeForm>some-other-thing</alternativeForm>
   -->
</nameEntry>
```

Though this of course would mean we'd need to check the content of each `<authorizedForm>` and `<alternativeForm>` and `<preferredForm>` in order to only created one `<conventionDeclaration>` for each rule/convention (here not a source!) that is mentioned in any of these.

**Comment by kerstarno** [comment]
As for the other topic: while this might be a real-world example, this doesn't necessarily mean we should "support" approaches like this by using them as illustrative examples here (or in the TL or the Best Practice Guide). It might mean that there actually is an indirect user need/request for `@encodingAnalog` in EAC-CPF, but as we have decided not to introduce this attribute, we might not want to highlight a work-around using an attribute that is not meant for this use case. Just saying :-)

**Comment by SJagodzinski** [comment]
> For the time being, at least, I have changed all IDREF attributes to IDREFS so that we can support the above example, 

@fordmadox : I agree

**Comment by SJagodzinski** [comment]
@fordmadox :

> > * Each `<nameEntry>` would get the appropriate `@status` "authorized" respectively "alternative" in EAC-CPF 2.0
> 
> I still need to re-write the transformation to utilize IDREFS instead of IDREF, but I don't understand the above rule.

 Why don't you understand this rule? A name is labelled as authorised or alternative. What is your question?

> `@status` can only have one value (authorized OR alternative). If a nameEntry is flagged as both in EAC 1.0, that should result in what? There's no way to bind the authorized reference to the nameEntry. E.g. if something like this were the input:
> 
> ```
>             <nameEntry scriptCode="Latn" transliteration="some-scheme">
>                 <part localType="100a">Pope, Alexander, </part>
>                 <part localType="100d">1688-1744 </part>
>                 <authorizedForm>lcnaf</authorizedForm>
>                 <alternativeForm>some-other-thing</alternativeForm>
>                 <alternativeForm>yet-another-thing</alternativeForm>
>                 <authorizedForm>why-not-another</authorizedForm>
>             </nameEntry>
> ```
> 
> What should the output be?

This example, and also the tl example, is one of the reasons for the users change request. This encoding might be correct for someone but it is not simple. But, why would we have to transform the status, if it is not clear and distinct? We could leave a comment like "unclear status authrorized or alternative form of name".

Referencing back to `<conventionDeclaration>`  for the rules in your example is quite convincing, also with @kerstarno [proposal for the rules ecoding](https://github.com/SAA-SDT/eac-cpf-schema/issues/117#issuecomment-762663563) in `<descriptiveNote>`.
